test/src/test/scala/DataSpec.scala

```diff
@@ -76,4 +77,12 @@ class DataSpec extends FlatSpec with MustMatchers with BeforeAndAfterAll {
       _.gender := Gender.MALE
     ))
   }
+
+  "RDD[Defaults]" should "have non-null default values after converting to Dataframe" in {
```

Comment:
I'd like to see two additional tests:
1. unset optional fields in proto2 should create nulls
2. unset primitive fields in proto3 should give default values.

Reply:
Sure, I'll rewrite the test to use proto2 and proto3 generated classes.